tests/redirection.py

```diff
@@ -0,0 +1,17 @@
+import nose
+import requests
+
+import fixture
+
+
+@nose.with_setup(fixture.start_tangelo, fixture.stop_tangelo)
+def test_redirect():
+    r = requests.get(fixture.url("redirect/redirect"))
+    assert r.ok
+    assert r.status_code == 200
+    assert r.text == "hello, world\n"
+
+    r = requests.get(fixture.url("redirect/internal_redirect"))
+    assert r.ok
+    assert r.status_code == 200
+    assert r.text == "hello, world\n"
```

Comment:
You should also test that the redirect sends a 303 and the internal_redirect doesn't, like so:

```
r = requests.get(fixture.url("redirect/redirect"), allow_redirects=False)
assert r.ok
assert r.status_code == 303
r = requests.get(fixture.url("redirect/internal_redirect"),
                 allow_redirects=False)
assert r.ok
assert r.status_code == 200
assert r.text == "hello, world\n"
````